app/models/schemas.py

```diff
@@ -4,7 +4,11 @@
 from enum import Enum
 from uuid import UUID, uuid4
 from datetime import datetime
-from energy.construction import EnergyConstructionOpaque, EnergyConstructionTransparent
+from .app.models.energy.construction import EnergyConstructionOpaque, EnergyConstructionTransparent
```

Comment:
```suggestion
from app.models.energy.construction import EnergyConstructionOpaque, EnergyConstructionTransparent
```

This change will fix the importing issue.

@tanushree04 `.` means the current directory. When you add it before app it is telling the module to look for an `app` module from where the file is and since it cannot find one it fails. It makes sense to add it before `energy` because the energy module lives in the same folder.